src/ontology/mondo-edit.obo

```diff
@@ -520167,6 +520139,52 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
+relationship: excluded_subClassOf MONDO:0001713 {source="Orphanet:3319", source="https://orcid.org/0000-0001-5208-3432"} ! inherited aplastic anemia
+relationship: has_characteristic MONDO:0021152 {source="OMIMPS:604498"} ! inherited
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6622" xsd:anyURI
+property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/640/congenital-amegakaryocytic-thrombocytopenia" xsd:anyURI {source="GARD:0000640"}
+
+[Term]
+id: MONDO:0800452
+name: congenital amegakaryocytic thrombocytopenia 1
+def: "A rare inherited bone marrow failure syndrome, in which the cause of the disease is a variation in the MPL gene. It is characterized by an isolated and severe decrease in the number of platelets and megakaryocytes during the first years of life that develops into bone marrow failure with pancytopenia later in childhood." [https://orcid.org/0000-0001-5208-3432, Orphanet:3319]
+subset: gard_rare {source="GARD:640"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:3319"}
+subset: orphanet_rare {source="Orphanet:3319"}
+subset: rare
+synonym: "amegakaryocytic thrombocytopenia, congenital" EXACT [MONDO:Lexical, OMIM:604498]
+synonym: "CAMT1" EXACT ABBREVIATION [DOID:0090118, MONDO:Lexical, OMIM:604498, Orphanet:3319]
+synonym: "congenital amegakaryocytic thrombocytopenic purpura" EXACT [DOID:0090118, Orphanet:3319]
+synonym: "thrombocytopenia congenital amegakaryocytic" RELATED [GARD:0000640]
+synonym: "thrombocytopenia, congenital amegakaryocytic" EXACT [OMIM:604498, OMIM:genemap2]
+xref: DOID:0090118 {source="MONDO:equivalentTo"}
+xref: GARD:640 {source="Orphanet:3319"}
+xref: ICD10CM:D61.0 {source="Orphanet:3319/attributed", source="Orphanet:3319/ntbt", source="Orphanet:3319", source="DOID:0090118"}
+xref: MESH:C535982 {source="Orphanet:3319/e", source="MONDO:equivalentTo", source="Orphanet:3319", source="DOID:0090118"}
+xref: NCIT:C115207 {source="MONDO:equivalentTo"}
+xref: OMIM:604498 {source="Orphanet:3319/e", source="MONDO:equivalentTo", source="Orphanet:3319", source="DOID:0090118"}
+xref: Orphanet:3319 {source="OMIM:604498", source="MONDO:equivalentTo", source="DOID:0090118"}
+xref: SCTID:716336002 {source="MONDO:equivalentTo"}
+xref: UMLS:C1327915 {source="OMIM:604498", source="Orphanet:3319/e", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="Orphanet:3319", source="DOID:0090118", source="NCIT:C115207"}
+is_a: MONDO:0800451 {source="OMIM:604498", source="https://orcid.org/0000-0001-5208-3432"} ! congenital amegakaryocytic thrombocytopenia
+intersection_of: MONDO:0800451 ! congenital amegakaryocytic thrombocytopenia
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7217 ! MPL
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
```

Comment:
An entry for [CAMT2](https://www.omim.org/entry/620481?search=camt2&highlight=camt2) _could_ be added vs. this exclude qc check.

Reply:
this will be added in the [OMIM migration](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/slurp/omim.tsv). @twhetzel or @sabrinatoro is someone assigned to do this? Is it on the horizon?